.cargo/config.toml

```diff
@@ -7,22 +7,16 @@
 ### INDIVIDUAL TEST AND LINT TASKS ###
 
 # Build and run all code paths except docs
-test-all = "test --all-features --all-targets"
+test-all = "make test-all-features"
```

Comment:
Nit: Instead of updating these aliases to invoke `cargo make`, please just delete them.  They were here previously to allow Makefile.toml and CI to execute the same command.  But now if CI invokes `cargo make` directly, we don't need these, and they will only cause confusion.

Reply:
Great, I was going to ask about this anyway :smile: 

I didn't want to break any workflows